daru-io.gemspec

```diff
@@ -35,4 +35,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'yard'
 
   spec.add_development_dependency 'guard-rspec' if RUBY_VERSION >= '2.2.5'
+
+  spec.add_runtime_dependency 'request-log-analyzer', '~> 1.13.4'
```

Comment:
Note that no other file format gem is required unconditionally: daru-io relies on user installing the proper gems. Otherwise, anybody who wants daru-io just for CSV import, will install TON of other gems for pdf, excel and whatnot. For example, look here: https://github.com/SciRuby/daru-io/blob/master/lib/daru/io/importers/excelx.rb#L12 — we have `optional_gem` construct for requiring something ONLY when it is used.